bin/alluxio-start.sh

```diff
@@ -52,8 +52,7 @@ check_mount_mode() {
     Mount);;
     SudoMount);;
     NoMount)
-      mount | grep "${ALLUXIO_RAM_FOLDER}" > /dev/null
-      if [[ $? -ne 0 || -z ${ALLUXIO_RAM_FOLDER} ]]; then
+      if [[ $(mount | grep -c ${ALLUXIO_RAM_FOLDER}) != 0 ]]; then
```

Comment:
@pfxuan 
I see what you want.  I think you need to change this alluxio.worker.data.folder to /userXYZ/alluxioworker/ .

If you really want to change the script to do that, I think you can check whether the parent directory  of ALLUXIO_RAM_FOLDER is mounted.

The current change you are making will not work if I set ALLUXIO_RAM_FOLDER to "/mnt/ramdisk" and /mnt/ramdisk is mounted manually.